src/district/server/web3_events.cljs

```diff
@@ -6,16 +6,49 @@
             [district.server.web3 :refer [web3]]
             [medley.core :as medley]
             [mount.core :as mount :refer [defstate]]
-            [taoensso.timbre :as log]))
+            [taoensso.timbre :as log]
+            [cljs-node-io.core :as io :refer [slurp spit]]
+            [clojure.edn :as edn]))
 
 (declare start)
 (declare stop)
 
+(def checkpoint-a
```

Comment:
I think cleaner would be if the atom is stored inside mount's web3-events atom. That's how we do it in other modules as well.

Reply:
It is called checkpoint and is inside web3-events state now.